boards/arm/frdm_k64f/doc/index.rst

```diff
@@ -378,3 +378,13 @@ of pyocd commands:
 
 .. _OpenSDA Serial and Debug Adapter:
    https://www.nxp.com/design/microcontrollers-developer-resources/ides-for-kinetis-mcus/opensda-serial-and-debug-adapter:OPENSDA#FRDM-K64F
+
+Experimental ENET Driver
+========================
+
+Current default ethernet driver is eth_mcux, with binding `nxp,kinetis-ethernet`. There is a new
+driver with binding `nxp,enet`, which is experimental and undergoing development, but will have
+enhanced capability, such as not hardcoding code for only one phy in the driver like eth_mcux.
+
+To build for this EVK with the new driver, include the experimental overlay to west build with
+the option `-DEXTRA_DTC_OVERLAY_FILE=nxp,enet-experimental.overlay`.
```

Comment:
can we add this option as a new test case in one of the network testing sample.yaml file

Reply:
ok